data_models/data_refinery_models/models.py

```diff
@@ -17,26 +18,101 @@ class Meta:
         abstract = True
 
 
-# This model still has a prototypical status, but I needed something to
-# test with and it's at least in the right ballpark
+class SurveyJob(TimeTrackedModel):
+    source_type = models.CharField(max_length=256)
+
+    # The start time of the query used to replicate
+    replication_started_at = models.DateTimeField(null=True)
+
+    # The end time of the query used to replicate
+    replication_ended_at = models.DateTimeField(null=True)
+
+    # The start time of the job
+    start_time = models.DateTimeField(null=True)
+
+    # The end time of the job
+    end_time = models.DateTimeField(null=True)
+
+    class Meta:
+        db_table = "survey_jobs"
+
+
+class BatchStatuses(Enum):
```

Comment:
If the download fails it will be recorded in the `status` field of the downloader_job. There will be code to check the statuses of jobs and retry the ones that fail. I don't think the batch needs to worry about this.